.github/workflows/ci-zap-dev.yml

```diff
@@ -0,0 +1,35 @@
+name: DEV - ZAP Scan
+on:
+  workflow_run:
+    workflows: ['Deployment to aws dev environment']
+    types:
+      - completed
+  push:
+    tags:
+      - security
+  pull_request:
+
```

Comment:
This script looks pretty similar to ci-zap.yml. What is it's purpose?

Reply:
Ahh, I thought the ci-zap action was just for prod, I'll remove this.